kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java

```diff
@@ -31,41 +31,44 @@
 
     private static final String TAG = "FeedbackInputFragment";
 
-    EditText editTextFeedbackComment;
-    Button buttonSubmitFeedback;
-    Button buttonCloseFragment;
+    private EditText editTextFeedbackComment;
+    private Button buttonSubmitFeedback;
+    private Button buttonCloseFragment;
 
-    FeedbackRatingGroup feedbackRatingGroup;
+    private FeedbackRatingGroup feedbackRatingGroup;
 
-    FeedbackFragmentListener feedbackFragmentListener;
+    private FeedbackFragmentListener feedbackFragmentListener;
 
-    public static final int RATING_POOR = 1;
-    public static final int RATING_AVERAGE = 2;
-    public static final int RATING_GOOD = 3;
+    public static final int RATING_POOR = 0;
+    public static final int RATING_POOR_VALUE = 1;
```

Comment:
do we need both? can we just keep 1, 5 and 10 constants?

Reply:
we need the indexes identify the level of ratings and to index the drawable arrays. the values of the ratings are seperate from their levels. Eg. rating of index one can have a value of 3 and index 2 can have a value of 7 etc

Reply:
used list, removed constant